src/index.ts

```diff
@@ -60,7 +60,14 @@ const configs: { recommended: TSESLint.Linter.Config & { plugins: string[] } } =
   recommended: { plugins: ['sonarjs'], rules: {} },
 };
 
-sonarjsRules.forEach(rule => (sonarjsRuleModules[rule[0]] = require(`./rules/${rule[0]}`)));
-sonarjsRules.forEach(rule => (configs.recommended.rules![`sonarjs/${rule[0]}`] = rule[1]));
+sonarjsRules.forEach(rule => (sonarjsRuleModules[rule] = require(`./rules/${rule}`)));
+sonarjsRules.forEach(rule => {
```

Comment:
since this forEach has a full body now, we probably can merge it with the previous one